Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-oversight.controller.ts

```diff
@@ -43,6 +43,8 @@
         oversigthOptions: Models.ViewModel.Generic.IMultipleSelectionWithSelect2ConfigViewModel<Models.Generic.NamedEntity.NamedEntityWithDescriptionAndExpirationStatusDTO>;
         oversightOptionsRemark: Models.ViewModel.Generic.IEditTextViewModel;
         isOversightCompleted: Models.ViewModel.Generic.ISingleSelectionWithFixedOptionsViewModel<Models.Api.Shared.YesNoUndecidedOption>;
+        oldIsOversightCompletedValue: Models.Api.Shared.YesNoUndecidedOption;
+        yesIsOversightCompletedValue = new Models.ViewModel.Shared.YesNoUndecidedOptions().options[1];
```

Comment:
I stedet for options[1] så lav en find eller filter der bruger enum til at udtrykke det valg du gerne vil sætte.. 
Husk at sætte den til private, eftersom den kun bruges i controlleren og ikke skal eksponeres til omverdenen